frontend/app/src/store/defi/liquity/actions.ts

```diff
@@ -72,9 +72,9 @@ export const actions: ActionTree<LiquityState, RotkehlchenState> = {
       }
     });
   },
-  async purge({ commit, rootGetters: { status } }) {
+  async purge({ commit }) {
     function resetStatus(section: Section) {
```

Comment:
looks like this also needs to be moved, there are also many in other file

Reply:
ok, let me check